src/components/Staking/StakingGuides.tsx

```diff
@@ -11,7 +11,7 @@ const StyledCardList = styled(CardList)`
   gap: 1rem;
 `
 
-const StakingGuides = () => {
+const StakingGuides: React.FC = () => {
```

Comment:
By convention, lets always define the `IProps` interface.

Reply:
OK